lm_eval/tasks/gpqa/cot_zeroshot/_gpqa_cot_zeroshot_yaml

```diff
@@ -0,0 +1,30 @@
+dataset_path: Idavidrein/gpqa
+group: gpqa
+output_type: generate_until
+process_docs: !function utils.process_docs
+training_split: train
+# Because huggingface dataset only has train split
+validation_split: train
+test_split: null
+doc_to_text: "What is the correct answer to this question:{{Question}}\nChoices:\n(A) {{choice1}}\n(B) {{choice2}}\n(C) {{choice3}}\n(D) {{choice4}}\nLet's think step by step: "
+doc_to_target: answer
+filter_list:
+  - name: "get-answer"
```

Comment:
I see you've included `MultiChoiceRegexFilter` in utils but it isn't used here or in the other tasks--intentional?